cpp/lib/src/ZoteroHarvesterConfig.cc

```diff
@@ -463,10 +461,12 @@ bool ParseExpectedLanguages(const std::string &expected_languages_string, Langua
     if (expected_languages_string.empty())
         return true;
 
-    // force? (deprecated, treat as invalid)
+    // force
     std::string expected_languages(expected_languages_string);
-    if (expected_languages[0] == '*')
-        return false;
+    if (expected_languages[0] == '*') {
+        language_params->force_detection_ = true;
+        expected_languages = expected_languages.substr(1);
```

Comment:
Is a construct like `*eng` admissible? I thought `*` was a standanlone. In the last case I don't get the `substr` construct.

Reply:
`*` has never been allowed standalone, it is only allowed together with one or more languages, like  `*eng` or  `*eng,fre,ger`.

Reply:
Ah, ok - thanks for the explanation.